gradle.properties

```diff
@@ -1,5 +1,5 @@
 # JVM args to run gradle
-org.gradle.jvmargs=-Xms1024m -Xmx2048m
+org.gradle.jvmargs=-Xms1024m -Xmx2048m -Dfile.encoding=UTF-8
```

Comment:
Why is this necessary? All file in the build are UTF8 per each project's settings https://github.com/bndtools/bnd/blob/79a9e4c4ff78c030a58efc381b6ce05f5cf2cb8e/biz.aQute.bnd.gradle/.settings/org.eclipse.core.resources.prefs#L2

and the .editorconfig https://github.com/bndtools/bnd/blob/79a9e4c4ff78c030a58efc381b6ce05f5cf2cb8e/.editorconfig#L13

and the Bnd gradle plugin https://github.com/bndtools/bnd/blob/79a9e4c4ff78c030a58efc381b6ce05f5cf2cb8e/biz.aQute.bnd.gradle/src/aQute/bnd/gradle/BndPlugin.groovy#L194 https://github.com/bndtools/bnd/blob/79a9e4c4ff78c030a58efc381b6ce05f5cf2cb8e/biz.aQute.bnd.gradle/src/aQute/bnd/gradle/BndPlugin.groovy#L203

I don't see that we need to do this yet again.

Reply:
I'll double check this. I was getting some error and/or seeing "?????" appear on the console for some of my non-ASCII string tests before I added this line. It is possible I didn't have the encoding set properly in my Eclipse files, though I didn't recall having the issue in Eclipse. It might be different now anyway because those tests are programmatically compiled at test execution time.

Reply:
Ok, if I don't include the UTF-8 flag in `gradle.properties`, then I get these failures in Gradle:

````

aQute.junit.platform.test.ActivatorTest > eclipseListener_reportsCustomNames_withOddCharacters FAILED
    org.opentest4j.MultipleFailuresError: Multiple Failures (2 failures)
        [[1] testWithNonASCII]
    Expecting:
     <"Prüfung 2">
    to be equal to:
     <"Pr▒fung 2">
    but was not.
    at ActivatorTest.eclipseListener_reportsCustomNames_withOddCharacters(ActivatorTest.java:410)
        [[2] testWithNonLatin]
    Expecting:
     <"Δοκιμή 3">
    to be equal to:
     <"?????? 3">
    but was not.
    at ActivatorTest.eclipseListener_reportsCustomNames_withOddCharacters(ActivatorTest.java:410)
````

However, the Travis build seems to be fine with it. I'll put it into my PR as I will be interested to see what the WindowsJDK8 build on Azure does.

Reply:
Ok, this has been fixed - see https://github.com/bndtools/bnd/pull/3145#issuecomment-511258972